local_dev_files/destroy_dev_env.sh

```diff
@@ -9,63 +9,33 @@ yellowColor='\033[0;33m'
-cd $basedir/env/cloud/ecr
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying DynamoDB${reset}\n"
-cd $basedir/env/cloud/dynamodb
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying S3${reset}\n"
-cd $basedir/env/cloud/s3
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying RDS${reset}\n"
-cd $basedir/env/cloud/rds
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying Redis${reset}\n"
-cd $basedir/env/cloud/redis
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Creating SNS queue${reset}\n"
-cd $basedir/env/cloud/sns
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Creating SQS queue${reset}\n"
-cd $basedir/env/cloud/sqs
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying Secrets Manager${reset}\n"
-cd $basedir/env/cloud/secrets
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying Network${reset}\n"
-cd $basedir/env/cloud/network
-# terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
-
-printf "${greenColor}...Destroying KMS${reset}\n"
-cd $basedir/env/cloud/kms
-terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn --terragrunt-config destroy_terragrunt.hcl
+terragrunt run-all destroy \
+    --non-interactive --log-level warn \
+    --queue-strict-include \
+    --working-dir $basedir/env \
+    --queue-include-dir $basedir/env/cloud/kms \
+    --queue-include-dir $basedir/env/cloud/network \
+    --queue-include-dir $basedir/env/cloud/secrets \
+    --queue-include-dir $basedir/env/cloud/sqs \
+    --queue-include-dir $basedir/env/cloud/s3 \
+    --queue-include-dir $basedir/env/cloud/ecr \
+    --queue-include-dir $basedir/env/cloud/sns \
+    --queue-include-dir $basedir/env/cloud/redis \
+    --queue-include-dir $basedir/env/cloud/rds \
+    --queue-include-dir $basedir/env/cloud/dynamodb \
+    --queue-include-dir $basedir/env/cloud/lambdas \
+    --queue-include-dir $basedir/env/cloud/vpn
```

Comment:
Are the `destroy_terragrunt.hcl` files still needed?

Reply:
No, I'll remove them.